pkg/sqs/service.go

```diff
@@ -183,7 +141,24 @@ func (s service) QueueExists(name string) (bool, error) {
 	return false, nil
 }
 
-func (s service) GetProperties(name string) (*Properties, error) {
+func (s *service) GetPropertiesByArn(arn string) (*Properties, error) {
```

Comment:
that looks a bit hacky, can we not instead get the queue url via its arn?

Reply:
Unluckily, that's not possible, without scanning all attributes for all existing queues.